pkg/gateway/model_build_targetgroup.go

```diff
@@ -390,11 +394,18 @@ func (t *latticeServiceModelBuildTask) buildTargetGroupSpec(
 		Namespace: namespace,
 		Name:      string(backendRef.Name()),
 	}
-	tgp, err := GetAttachedPolicy(ctx, t.client, refObjNamespacedName, &anv1alpha1.TargetGroupPolicy{})
-
+	tgps, err := GetAttachedPolicies(ctx, t.client, refObjNamespacedName, &anv1alpha1.TargetGroupPolicy{})
 	if err != nil {
 		return model.TargetGroupSpec{}, err
 	}
+	var tgp *anv1alpha1.TargetGroupPolicy
+	if len(tgps) >= 1 {
+		tgp = tgps[0]
+		if len(tgps) > 1 {
+			t.log.Errorf("More than one TargetGroupPolicy is attached to the k8sService %s, "+
+				"only the first one TargetGroupPolicy [%s/%s] will take effect, other TargetGroupPolicies will be ignored", refObjNamespacedName, tgp.Namespace, tgp.Name)
+		}
+	}
```

Comment:
Ideally we should do that behaviour: `If there are two tgps  attached to a svc(serviceExport), set one policy status to accepted and set another one status to conflicted.`

But that need more extra work considering our current code structure (GetTargetGroupPolicy logic exists in the giant route reconcile logic). We could impliment that behaviour in a seperate PR.

For now, we just keep old code and new code have same behaviour: just make the first tgp(same for VpcAssociationPolicy) take effect and ignore other same kind policies (print a Error for it)